ReactWindows/ReactNative.Shared/Views/TextInput/ReactTextBox.cs

```diff
@@ -20,6 +21,35 @@ class ReactTextBox : TextBox
         private int _eventCount;
         private bool _selectionChangedSubscribed;
         private bool _sizeChangedSubscribed;
+        private ClearButtonModeType _clearButtonMode = ClearButtonModeType.Default;
+
+        public ClearButtonModeType ClearButtonMode
+        {
+            get
+            {
+                return _clearButtonMode;
+            }
+            set
+            {
+                if (_clearButtonMode != value)
+                {
+                    _clearButtonMode = value;
+                    UpdateDeleteButtonVisibility();
+                }
+            }
+        }
+
+        private Button DeleteButton
```

Comment:
a private property is tantamount to declaring var directly, usually we don't bother with properties, just declare the var